source/includes/roles/_30-create.md.erb

```diff
@@ -0,0 +1,139 @@
+## Create a gocd role
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/security/roles' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.roles %>' \
+      -H 'Content-Type: application/json' \
+      -X POST -d '{
+         "name": "new-role",
+         "type": "gocd",
+         "attributes" : {
+          "users": ["bob"]
+         }
+        }'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.roles %>; charset=utf-8
+ETag: "cbc5f2d5b9c13a2cc1b1efb3d8a6155d"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/security/roles/new-role"
+    },
+    "doc": {
+      "href": "https://api.gocd.io/#roles"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/security/roles/:role_name"
+    }
+  },
+  "name": "new-role",
+  "type": "gocd",
+  "attributes": {
+    "users": ["bob"]
+  }
+}
+```
+
+Create a gocd role. A gocd role is a group of users having same permissions in go.
```

Comment:
There is link for `create plugin role` in left panel.

![screen shot 2017-05-22 at 10 18 55 am](https://cloud.githubusercontent.com/assets/7871209/26293510/e6ff7f56-3eda-11e7-885a-78b1075a7b85.png)